File: ts/wam/7506.ts
export const WamWebcWebtpPdfViewer = 7506 as const

export interface WebcWebtpPdfViewerEvent {
  /** field 1, wire `webtp_error_code` */
  webtpErrorCode?: string
  /** field 2, wire `webtp_error_message` */
  webtpErrorMessage?: string
  /** field 3, wire `webtp_error_stack` */
  webtpErrorStack?: string
  /** field 4, wire `webtp_error_type` */
  webtpErrorType?: string
  /** field 5, wire `webtp_event` */
  webtpEvent?: WEBTPEVENTTYPE
  /** field 6, wire `webtp_file_size` */
  webtpFileSize?: string
  /** field 7, wire `webtp_sdk_version` */
  webtpSdkVersion?: string
  /** field 8, wire `webtp_session_id` */
  webtpSessionId?: string
  /** field 9, wire `webtp_source` */
  webtpSource?: WEBTPSOURCETYPE
  /** field 10, wire `webtp_telemetry_data` */
  webtpTelemetryData?: string
}

File: go/wam/7506.go
const WamWebcWebtpPdfViewer = 7506 // channel: regular

type WebcWebtpPdfViewerEvent struct {
	WebtpErrorCode string `wam:"webtp_error_code"` // field 1
	WebtpErrorMessage string `wam:"webtp_error_message"` // field 2
	WebtpErrorStack string `wam:"webtp_error_stack"` // field 3
	WebtpErrorType string `wam:"webtp_error_type"` // field 4
	WebtpEvent WEBTPEVENTTYPE `wam:"webtp_event"` // field 5
	WebtpFileSize string `wam:"webtp_file_size"` // field 6
	WebtpSdkVersion string `wam:"webtp_sdk_version"` // field 7
	WebtpSessionId string `wam:"webtp_session_id"` // field 8
	WebtpSource WEBTPSOURCETYPE `wam:"webtp_source"` // field 9
	WebtpTelemetryData string `wam:"webtp_telemetry_data"` // field 10
}